### DISCUSSION TITLE
分享润的资料

### DISCUSSION THREAD

**Discussion by skyabcher** [The-Run-Philosophy-Organization/run#391] [General]
这是我平时收集的关于润的资料合集，包括在澳洲的护士nika跟立党的知乎问答，在美国写代码的孙玲，在日本做生意的老蔡等故事合集，pdf跟epub格式的，现在分享给大家，我不确定这些资料能不能放到仓库，所以只能用百度网盘分享。
https://pan.baidu.com/s/1nIziHkVeqdwqfUjnmwFQRQ?pwd=8ses

**Comment by the-run-philosophy** [comment]
其中有一些pdf可以变成text

**Comment by the-run-philosophy** [comment]
@laijiawei  do you mind adding some of the text content first?

**Comment by sece1024** [comment]
不如放Google云上？

**Comment by skyabcher** [comment]
> 不如放Google云上？
文件太大了，上传速度太慢了，放不了

**Comment by gao5526823** [comment]
打开看了一下，非常实用，非常感谢您的分享。我本人比较穷，正在准备加拿大的，听说加拿大和日本比较容易，一起加油！

**Comment by Achilles020** [comment]
thx for your shearing

**Comment by Mazakanull** [comment]
謝整理

**Comment by biensuur** [comment]
红豆泥阿里嘎多！
